iroh-bytes/src/protocol/range_spec.rs

```diff
@@ -97,27 +112,45 @@ impl fmt::Debug for RangeSpec {
     }
 }
 
-/// A compressed sequence of range specs
+/// A chunk range specification for a collection of blobs.
```

Comment:
"sequence of blobs"? You could use this for specifying ranges for any sequence of blobs, and calling it a sequence makes it clear why the thing is called RangeSpec *Seq* ?

Reply:
+1